cypress.config.js

```diff
@@ -1,16 +1,24 @@
 const { defineConfig } = require('cypress');
+const faker = require('faker');
 
 module.exports = defineConfig({
   e2e: {
+    viewportHeight: 1100,
+    viewportWidth: 1320,
     baseUrl: 'https://www.demoblaze.com/',
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + `${randomNumber}`,
-            email: 'test' + `${randomNumber}` + '@mail.com',
-            password: 'Password12345!'
+            username: faker.name.firstName(),
+            // country: faker.location.country(),
+            country: 'Ukraine',
+            // city: faker.location.city(),
```

Comment:
Commented code found. Please remove the commented code if not required.

Reply:
Done